setup/ubuntu14.04/ansible-playbook.yaml

```diff
@@ -1,7 +1,6 @@
 ---
 - hosts: iojs-build-ubuntu1404
-
-  remote_user: root
+  sudo: yes
```

Comment:
We have a few similar cases in other os/provider combos. In my `ansible 2.0` refactor I've solved this by doing exceptions based on above pair. We can either solve it through the inventory file or through logic in handlers. As for now; you should be able to override with `host_vars`?